Backend/templates/gitlab-ci/example-gitlab-ci.yml

```diff
@@ -26,15 +26,26 @@ TestMR:
     - echo 'Tests ✅'
 
 
-###############
-# Master jobs #
-###############
+##############
+# Build jobs #
+##############
 
-Build:
-  extends: .build_template
+BuildDev:
+  extends: .build_dev_template
   script:
     - TODO Insert code to build and generate the web service artifact here
   artifacts:
-    expire_in: 1 hour
+    expire_in: 1 hour # TODO validate this
     paths:
-      - TODO insert artifact name here
+      - TODO insert artifact name here to be used in Dockerfile
+
+  
+BuildProd:
+  extends: .build_prod_template
+  script:
+    - TODO Insert code to build and generate the web service artifact here
```

Comment:
instead of this, we could call a script here, no? what do you think?

Reply:
Fine by me

This file is the only one needed in the service repo, so if a script is better, we should use it 🚀